GVRf/Framework/src/org/gearvrf/GVRMaterial.java

```diff
@@ -267,6 +268,8 @@ public GVRTexture getTexture(String key) {
     }
 
     public void setTexture(String key, GVRTexture texture) {
+        checkStringNotNullOrEmpty("key", key);
+        checkNotNull(GVRTexture.class.getSimpleName(), texture);
```

Comment:
Parameter name, not parameter type. That is, `"texture"` not `GVRTexture.class.getSimpleName()`.